### DISCUSSION TITLE
Support systemd `sd_notify`

### DISCUSSION THREAD

**Discussion by solracsf** [juicedata/juicefs#2810] [Ideas]
I have a systemd service to start/stop juicefs mount, but it only works in [simple](https://www.freedesktop.org/software/systemd/man/systemd.service.html#Examples) mode, not `notify`.
Is https://www.freedesktop.org/software/systemd/man/sd_notify.html implemented (or can it be)?

**Comment by SandyXSD** [comment]
No, it's not implemented yet.
